### ISSUE TITLE
input button and link button

### ISSUE THREAD

**Issue by oupala** [jgthms/bulma#2546]
### Overview of the problem

This is about the Bulma **CSS framework**

I'm using Bulma **version** [0.7.5]
My **browser** is: Firefox 67.0.4

I am *pretty* sure this issue is **not a duplicate**

### Description

Input buttons and link buttons are not of the same size. It makes it hard to mix them in a group of buttons.

### Steps to Reproduce

See this [jsfiddle](https://jsfiddle.net/9tqvzr8j/2/).

### Expected behavior

In my opinion, `input class="button` and `a class="button` should be of the same size.

### Actual behavior

In fact, input button is 13x18 while class button is 7x18.

**Comment by VVoruganti** [comment]
This seems to change depending on the browser. In chrome the input button was 9x18

**Comment by oupala** [comment]
You're right. The button size depends on its content. I changed the content to add a simple `A` letter into all the buttons.

- `input` button is 9x18
- `a` link button is 7x18

**Comment by oupala** [comment]
The issue is still open, from my point of view.

**Comment by oupala** [comment]
The issue is still open, from my point of view.

**Comment by oupala** [comment]
The issue is still open, from my point of view.

I don't have any idea on how to make it go further.

**Comment by oupala** [comment]
The issue is still open, from my point of view.

I don't have any idea on how to make it go further.
